blazor/dropdown-list/value-binding.md

```diff
@@ -0,0 +1,216 @@
+
+The following code demonstrates, Value and Text field of the DropDownList component For instance, the selected item is `Badminton` (Text Field i.e., Game) but the value field holds `Game2` (Value Field i.e., ID).
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/text-and-value.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Text and Value](./images/value-binding/blazor_dropdown_text-and-value.png)
+
+## Primitive type binding
+
+The DropDownList has support to load array of primitive data such as strings and numbers.
+
+The following code demonstrates array of string as datasource to the DropDownList component.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/primitive-type-string %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Primitive Type as string](./images/value-binding/blazor_dropdown_primitive-type-string.png)
+
+The following code demonstrates array of int as datasource to the DropDownList component.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/primitive-type-int %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Primitive Type as int](./images/value-binding/blazor_dropdown_primitive-type-int.png)
+
+## Object binding
+
+object data binding refers a memory location where object data is stored.
+
+In the following example, `Game` column has been mapped to the [`DropDownListFieldSettings.Value`](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.FieldSettingsModel.html#Syncfusion_Blazor_DropDowns_FieldSettingsModel_Value).
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/object-binding.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with object values](./images/value-binding/blazor_dropdown_object-binding.png)
+
+## Enum binding
+
+You can bind enum data to DropDownList component. The following code helps you get a string value from the enumeration data.
+
+{% highlight cshtml %}
+
+{% include_relative code-snippet/value-binding/enum-binding.razor %}
+
+{% endhighlight %}
+
+![Blazor DropDownList with Enum Data](./images/value-binding/blazor_dropdown_enum-binding.png)
+
+## Show or hide clear button
+
+You can use [ShowClearButton](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_ShowClearButton) property to specify whether to show or hide the clear button. When the clear button is clicked, `Value`, `Text`, and `Index` properties are reset to null.

```

Comment:
Use the [ShowClearButton](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.DropDowns.SfDropDownList-2.html#Syncfusion_Blazor_DropDowns_SfDropDownList_2_ShowClearButton) property to specify whether to show or hide the clear button. When the clear button is clicked, the `Value,` `Text,` and `Index` properties are reset to null.